charts/xapp-hello-world/templates/deployment.yaml

```diff
@@ -0,0 +1,130 @@
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "xapp_core.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "xapp_core.labels" . | nindent 8 }}
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      serviceAccountName: {{ include "xapp_core.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{ if or (.Values.xappEndpoints) (.Values.developerMode.enabled) }}
+      initContainers:
+        {{ if .Values.xappEndpoints }}
+        {{ if .Values.xappEndpoints.REDIS_URL }}
+        - name: check-redis
+          image: "{{ .Values.initImage.repository }}:{{ .Values.initImage.tag }}"
+          imagePullPolicy: {{ .Values.initImage.pullPolicy }}
+          command:
+            - "/bin/sh"
+            - "-c"
+            - |
+                until [ "$(timeout 10s redis-cli -u {{ tpl .Values.xappEndpoints.REDIS_URL . }} ping)" == "PONG" ]
+                do
+                  echo "$(date) Waiting for redis"
+                  sleep 1
+                done
+                echo "$(date) Redis ready"
+        {{ end }}
+        {{ if .Values.xappFrameworkConfig.flushOnDeployment }}
+        - name: flush-redis
+          image: "{{ .Values.initImage.repository }}:{{ .Values.initImage.tag }}"
+          imagePullPolicy: {{ .Values.initImage.pullPolicy }}
+          command:
+            - "/bin/sh"
+            - "-c"
+            - "redis-cli -u {{ tpl .Values.xappEndpoints.REDIS_URL . }} FLUSHALL"
+        {{ end }}
+        {{ if .Values.xappEndpoints.KAFKA_URL }}
+        - name: check-kafka
+          image: "{{ .Values.initImage.repository }}:{{ .Values.initImage.tag }}"
+          imagePullPolicy: {{ .Values.initImage.pullPolicy }}
+          command:
+            - "/bin/sh"
+            - "-c"
+            - |
+                KAFKA_URL="{{ tpl .Values.xappEndpoints.KAFKA_URL . | trimPrefix "kafka://" }}"
+                until kcat -b "$KAFKA_URL" -t kafka-cluster-init -C -o -1 -e
+                do
+                  echo "$(date) Waiting for kafka"
+                  sleep 1
+                done
+                echo "$(date) Kafka ready"
+        {{ end }}
+        {{ end }}
+      {{ end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{ if .Values.developerMode.enabled }}
+          command:
+            - "/bin/bash"
+            - "-c"
+            - |
+              sudo service ssh start
+              echo 'xapp:{{ .Values.developerMode.devPassword }}' | sudo chpasswd
+              git init /home/xapp/xapp/core/
```

Comment:
This should  be git init /home/xapp/xapp/

Reply:
Done (together with a rebase of `main`)